_includes/releases/cloud/2023-05-01.md

```diff
@@ -10,21 +10,21 @@ The following changes were made to {{ site.data.products.serverless }} for **new
 - All resources are available instantly at the beginning of each month (burst and baseline RUs are now deprecated).
 - You can now [set separate RU and storage limits](../cockroachcloud/serverless-cluster-management.html#edit-your-resource-limits) for your clusters, or set a spend limit that will be divided between resources automatically.
 - You can now [set unlimited resources](../cockroachcloud/serverless-cluster-management.html#edit-your-resource-limits) for a cluster so your workload is never restricted.
-- Organizations on the free plan will have at most one free Serverless cluster (instead of the previous limit of five free clusters). **We will continue to maintain all existing free clusters with the new organization-level free resources**. 
-- Organizations can now create 200 Serverless clusters instead of the previous limit of five total clusters.
+- Organizations on the free plan will have at most one free Serverless cluster (instead of the previous limit of five free clusters). **Cockroach Labs will continue to maintain all existing free clusters with the new organization-level free resources**. 
```

Comment:
```suggestion
- Organizations on the free plan will have at most one free serverless cluster (instead of the previous limit of five free clusters). **Cockroach Labs will continue to maintain all existing free clusters with the new organization-level free resources**. 
```